frontend/src/pages/TermPlanner/TermPlanner.tsx

```diff
@@ -36,6 +36,8 @@ const TermPlanner = () => {
 
   const [termsOffered, setTermsOffered] = useState<Term[]>([]);
   const [isDragging, setIsDragging] = useState(false);
+  const [checkYearMultiTerm, setCheckYearMultiTerm] = useState('');
```

Comment:
So, I can't remove the `checkYearmultiTerm` since `checkYearmultiTerm` helps to check which draggable course in the term box we are dragging and then adds a badge to that course. If I remove `checkYearmultiTerm` it will add a badge to all the courses in that term box.